src/main/java/com/massivecraft/factions/cmd/CmdShow.java

```diff
@@ -69,18 +76,18 @@ public void perform() {
             double refund = value * Conf.econClaimRefundMultiplier;
             if (value > 0.0D) {
                 String stringValue = Econ.moneyString(value);
-                String stringRefund = refund > 0.0D ? " (" + Econ.moneyString(refund) + " depreciated)" : "";
-                msg("<a>Total land value: <i>" + stringValue + stringRefund);
+                String stringRefund = (refund > 0.0) ? (TL.COMMAND_SHOW_DEPRECIATED.format(Econ.moneyString(refund))) : "";
+                msg(TL.COMMAND_SHOW_LANDVALUE, stringValue, stringRefund);
             }
 
             if (Conf.bankEnabled) {
-                msg("<a>Bank contains: <i>" + Econ.moneyString(Econ.getBalance(faction.getAccountId())));
+                msg(TL.COMMAND_SHOW_BANKCONTAINS, Econ.moneyString(Econ.getBalance(faction.getAccountId())));
             }
 
         }
 
-        String allyList = p.txt.parse("<a>Allies: ");
-        String enemyList = p.txt.parse("<a>Enemies: ");
+        String allyList = p.txt.parse("<a>" + TL.COMMAND_SHOW_ALLIES);
+        String enemyList = p.txt.parse("<a>" + TL.COMMAND_SHOW_ENEMIES);
```

Comment:
Its for the <a> color (previously .color gold) currently on mobile and I
don't have access to my pc, gonna do it tomorrow if you haven't merged &
edited it yourself.
Op 3-jan.-2015 23:30 schreef "Trent Hensler" <notifications@github.com>:

> In src/main/java/com/massivecraft/factions/cmd/CmdShow.java
> <https://github.com/drtshock/Factions/pull/203#discussion-diff-22435178>:
>
> >              }
> >
> >          }
> >
> > -        String allyList = p.txt.parse("<a>Allies: ");
> > -        String enemyList = p.txt.parse("<a>Enemies: ");
> > +        String allyList = p.txt.parse("<a>" + TL.COMMAND_SHOW_ALLIES);
> > +        String enemyList = p.txt.parse("<a>" + TL.COMMAND_SHOW_ENEMIES);
>
> We wouldn't need to have the text parsing as well as the if we are using
> TL.
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/drtshock/Factions/pull/203/files#r22435178>.
>